cmd/rpcdaemon/cli/config.go

```diff
@@ -61,6 +62,7 @@ func RootCommand() (*cobra.Command, *Flags) {
 	rootCmd.PersistentFlags().StringSliceVar(&cfg.API, "http.api", []string{"eth"}, "API's offered over the HTTP-RPC interface")
 	rootCmd.PersistentFlags().Uint64Var(&cfg.Gascap, "rpc.gascap", 0, "Sets a cap on gas that can be used in eth_call/estimateGas")
 	rootCmd.PersistentFlags().Uint64Var(&cfg.MaxTraces, "trace.maxtraces", 200, "Sets a limit on traces that can be returned in trace_filter")
+	rootCmd.PersistentFlags().StringVar(&cfg.TraceType, "trace.type", "parity", "Specify the type of tracing [geth|parity*] (experimental)")
```

Comment:
If this flag is only supposed to make a difference for `trace_` methods, then I would not even add it, and always use `parity` traces. I do not believe anyone would want to use `trace_` methods (they don't exist in geth) and expect `geth` format

Reply:
I'm using it mostly for my own debugging. Sometimes I need the original Geth traces, and this allows me to see them without having to re-build. It will probably be removed eventually, but I can also envision using it for an 'enhanced' mode where we correct some incorrect stuff from the Parity traces.